cut-n-paste/zoom-control/ephy-zoom.h

```diff
@@ -72,11 +72,9 @@ static const guint n_zoom_levels = G_N_ELEMENTS (zoom_levels);
 #define ZOOM_IN		(-1.0)
 #define ZOOM_OUT	(-2.0)
 
-guint	ephy_zoom_get_zoom_level_index	 (float level);
+guint	ephy_zoom_get_zoom_level_index	 (double level);
 
-float	ephy_zoom_get_changed_zoom_level (float level, gint steps);
-
```

Comment:
Why `ephy_zoom_get_zoom_level_index` is removed here?

Reply:
The function `ephy_zoom_get_changed_zoom_level` is used only in ephy-zoom.c after its implementation.
```
$ git grep -n ephy_zoom_get_changed_zoom_level
cut-n-paste/zoom-control/ephy-zoom.c:59:ephy_zoom_get_changed_zoom_level (double level, gint steps)
cut-n-paste/zoom-control/ephy-zoom.c:70:        return ephy_zoom_get_changed_zoom_level (level, 0);
```

Reply:
Good.